classes/settings_provider.php

```diff
@@ -148,18 +148,17 @@ public static function get_requiresafeexambrowser_options($context = null) : arr
         $options[self::USE_SEB_NO] = get_string('no');
         $options[self::USE_SEB_CONFIG_MANUALLY] = get_string('seb_use_manually', 'quizaccess_seb');
 
-        // @codingStandardsIgnoreStart
-        // TODO: Implement following features and uncomment options.
-        //if ($context && has_capability('quizaccess/seb:manage_seb_sebconfigtemplate', $context)) {
-        //    $options[self::USE_SEB_TEMPLATE] = get_string('seb_use_template', 'quizaccess_seb');
-        //}
+        if ($context && has_capability('quizaccess/seb:manage_seb_templateid', $context)) {
+            if (template::get_records(['enabled' => 1])) {
```

Comment:
How about rewrite get_template_options and return an empty array instead if there are no available templates? and then use get_template_options here to find out a number of active templates? Also would be good to make this method protected or cover it with tests.